cuda_core/tests/memory_ipc/conftest.py

```diff
@@ -0,0 +1,33 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+from cuda.core.experimental import Device, DeviceMemoryResource
+
+POOL_SIZE = 2097152
+
+
+@pytest.fixture(scope="function")
```

Comment:
```suggestion
@pytest.fixture
```

This is the default behavior in pytest.